src/main/java/mod/gate/core/mixin/ScoreboardPacketMixin.java

```diff
@@ -0,0 +1,30 @@
+package mod.gate.core.mixin;
+
+import mod.gate.core.events.EventHandler;
+import mod.gate.core.events.ingamehud.ScoreboardUpdateEvent;
+import mod.gate.utils.Scoreboard;
+import net.minecraft.client.network.ClientPlayNetworkHandler;
+import net.minecraft.network.packet.s2c.play.TeamS2CPacket;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(ClientPlayNetworkHandler.class)
+public class ScoreboardPacketMixin {
+    private long lastUpdateAt = 0;
+
+    @Inject(at= @At("TAIL"), method = "onTeam")
```

Comment:
Why do you inject at tail?

Reply:
was to do it once every change of scoreboard was done